internal/querynode/segment_loader.go

```diff
@@ -42,12 +42,14 @@ import (
 	"github.com/milvus-io/milvus/internal/types"
 	"github.com/milvus-io/milvus/internal/util/concurrency"
 	"github.com/milvus-io/milvus/internal/util/funcutil"
+	"github.com/milvus-io/milvus/internal/util/indexparamcheck"
 	"github.com/milvus-io/milvus/internal/util/metricsinfo"
 	"github.com/milvus-io/milvus/internal/util/timerecord"
 )
 
 const (
 	requestConcurrencyLevelLimit = 8
+	UsedDiskMemoryRatio          = 6
```

Comment:
move it to configuration ?

Reply:
This parameter does not want to be exposed to the configuration, and this parameter is bound to the diskann built-in strategy of knowhere. It is not helpful to move to configuration.